src/main/java/org/springframework/data/aerospike/cache/AerospikeCacheManager.java

```diff
@@ -63,9 +63,10 @@
 	 * set name "aerospike".
 	 * 
 	 * @param aerospikeClient the {@link AerospikeClient} instance.
+	 * @param aerospikeConverter
 	 */
-	public AerospikeCacheManager(AerospikeClient aerospikeClient) {
-		this(aerospikeClient, Collections.<String>emptyList());
+	public AerospikeCacheManager(AerospikeClient aerospikeClient, MappingAerospikeConverter aerospikeConverter) {
```

Comment:
backward compatibility for 1.0.0-snapshot which is not even published in maven repo? really? :)